### ISSUE TITLE
BSD-3-Clause license "violation"

### ISSUE THREAD

**Issue by kapouer** [sqlcipher/sqlcipher#307]
Hi,

the BSD-3-Clause license states:
```
- Redistributions of source code must retain the above copyright notice...
```
which is for sqlite3:
```
Copyright (c) MapBox
All rights reserved.
```
which isn't in https://github.com/sqlcipher/sqlcipher/blob/fda4c68bb474da7e955be07a2b807bda1bb19bd2/LICENSE#L1-L2

The proper way to fix this is to simply retain the original copyright line below or above your copyright line.

**Comment by sjlombardo** [comment]
Hello @kapouer, thanks for getting in touch. I'm somewhat confused by this issue. SQLCipher has no dependencies on MapBox at all. If MapBox is using SQLCipher (which it seems like they might be), then their project should be including our SQLCipher copyright statement, we the other way around. In that case, this Issue should be filed against their project, since they would be the ones in violation of the license.

In the event that you meant SQLite instead of MapBox, that situation is different. SQLite is not distributed under the BSD-3 clause license, it is [in the public domain](https://www.sqlite.org/copyright.html). As a result, there is no license to abide by. Even so, we do extensively communicate that SQLCipher is based on SQLite in all project documentation (e.g. the [project website](https://www.zetetic.net/sqlcipher/), [README](https://github.com/sqlcipher/sqlcipher/blob/master/README.md), [Open Source Legal Notices](https://www.zetetic.net/sqlcipher/license/).

So, to summarize, SQLCipher is licensed under the 3-clause BSD, but SQLite is not. Projects that use SQLCipher Community Edition under that license are require to adhere to SQLCipher license and attribute SQLCipher accordingly. Projects implementing SQLCipher (or SQLite) are not required to attribute SQLite since it is public domain. However, SQLCIpher does extensively credit SQLite outside of the license text in recognition of the importance of that project as the core library that SQLCipher is based on.

Let me know if that clears things up, or if there is something that we are missing.

**Comment by kapouer** [comment]
Oops ! I had an ongoing issue about node-sqlite and confused that one with sqlite3. Sorry !
